x/stake/genesis.go

```diff
@@ -32,7 +33,7 @@ func DefaultGenesisState() GenesisState {
 }
 
 // InitGenesis - store genesis parameters
-func InitGenesis(ctx sdk.Context, k Keeper, data GenesisState) {
+func InitGenesis(ctx sdk.Context, k Keeper, data GenesisState) (vals []abci.Validator) {
```

Comment:
Is there a reason to use a named return var when we're not using that actual named variable?